src/r2mm/manager/PackageDexieStore.ts

```diff
@@ -174,3 +184,24 @@ export async function isLatestPackageListIndex(community: string, hash: string)
 export async function setLatestPackageListIndex(community: string, hash: string) {
     await db.indexHashes.put({community, hash, date_updated: new Date()});
 }
+
+async function getDexiePackage(community: string, packageName: string) {
```

Comment:
Not a huge fan of this naming. It makes it sound Dexie related rather than data related

Reply:
Note that `getDexiePackage` and `getDexieVersion` aren't exported, so I'd consider them internal implementation details of the module. The naming refers to the (also internal) interfaces of the same name. That being said, I'm open for suggestion for better names for both the interfaces and the functions.

Reply:
Didn't change the naming as no alternatives were provided.

Reply:
getPackage? getPackageFromDatabase?

Reply:
Renamed [the function](https://github.com/ebkr/r2modmanPlus/pull/1485/files#diff-2122463b05e5c9342979622ca58d48970d706008d1d9df423063e69e07055988R190).